src/ui/ui_text_box.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "ui_container.h"
+#include "ui_text_node.h"
+#include "ui_rectangle.h"
+#include "util/color.h"
+#include <string>
+
+class UITextBox : public UIContainer {
+public:
+    UITextBox() {} // default ctor
```

Comment:
Is it wise to provide this? May be better to force usage of the parameterized constructor.